javascript/inavapp.js

```diff
@@ -1,10 +1,7 @@
-(function() {
-  var app = angular.module("inavApp", ['ngRoute']);
```

Comment:
The big crux of the error you were seeing was actually due to this. 

Seeing as you're using Angular 1.5.0, ngRoute is a separate dependency and requires you to include another js script in your file. Removing it resolved the error and is okay to do seeing as you weren't using it. 

However, if you'd like to use ngRoute, you can see how to include that script and using Routing & Multiple Views via the Angular documentation. 

https://docs.angularjs.org/tutorial/step_07